docs/Toy/examples/ast.toy

```diff
@@ -0,0 +1,76 @@
+# RUN: toyc-ch1 %s -emit=ast 2>&1 | FileCheck %s
```

Comment:
Can't that be a test?

Reply:
They test them in MLIR, but I don't think we need to. At least, not any more than the notebooks running with no errors, which the CI already tells us

Reply:
But the notebook in this PR is not in CI right?